### PR TITLE
Chores/hide empty optional listing data #140694317

### PR DESCRIPTION
**Pull request by chrisdolendo**
Still waiting on city but keeping this on radar.
To do: merge https://github.com/Exygy/sf-dahlia-web/pull/922 since it includes the `::` optimizations.

### CODE REVIEW

File: app/assets/javascripts/listings/templates/listing/_panel-apply-leasing-agent.html.slim [hunk 1]
```diff
@@ -1,8 +1,9 @@
-p.content-group_address.c-steel
-  | {{ listing.Leasing_Agent_Street }}
-  br
-  | {{ listing.Leasing_Agent_City }} {{ listing.Leasing_Agent_State }} {{ listing.Leasing_Agent_Zip }}
-a.t-small.d-block.ui-link.margin-bottom.hide-for-print translate="LABEL.VIEW_MAP" title="{{ 'LABEL.VIEW_MAP' | translate }}" target="_blank" ng-href="https://www.google.com/maps/place/{{formattedLeasingAgentAddress(listing)}}"
+span ng-if="listing.Leasing_Agent_Street"
```
Comment (kaptron) [inline app/assets/javascripts/listings/templates/listing/_panel-apply-leasing-agent.html.slim]: don't use `span` to contain other block elements like this, in that case always use a `div`

Comment (chrisdolendo) [reply app/assets/javascripts/listings/templates/listing/_panel-apply-leasing-agent.html.slim]: @kaptron updated :)

**Comment by chrisdolendo** [comment]
@kaptron thanks for the review! updated :)

**Comment by chrisdolendo** [comment]
@kaptron btw i might re-request your review or re-open this PR because it's still WIP and I'm waiting for the city for their final list of optional data :)

**Comment by chrisdolendo** [comment]
closing this PR, keeping branch for when the City is ready to move forward with the story.

**Pull request closed without merging**